cypress/integration/admin/pageBuilder/page/publishPage.spec.js

```diff
@@ -1,97 +1,113 @@
-//                 cy.findByTestId("confirmationdialog-confirm-action").click();
-//             });
-//         });
-//
-//         it.skip(`Step 4: Check updated page title in page preview`, () => {
-//             return cy.pbListPages({ limit: 1, search: { query: pageTitle2 } }).then(([page]) => {
-//                 const { path } = page;
-//                 cy.visit(`${Cypress.env("WEBSITE_URL")}${path}`);
-//
-//                 cy.title().should("contain", pageTitle2);
-//             });
-//         });
-//
-//         it.skip(`Step 5: Delete page immediately`, () => {
-//             cy.visit("/page-builder/pages");
-//             return cy.pbListPages({ limit: 1, search: { query: pageTitle2 } }).then(([page]) => {
-//                 const { id } = page;
-//                 // Delete page by deleting first revision
-//                 cy.pbDeletePage({ id: `${id.slice(0, -1)}1` });
-//             });
-//         });
-//     }
-// );
+import uniqid from "uniqid";
+
+context(
+    "Should be able to publish and re-publish a page, and see changes on the public site",
+    () => {
+        let id;
+        const pageTitle1 = `Test published page 1`;
+        const pageTitle2 = `Test published page 2`;
+
+        beforeEach(() => {
+            id = uniqid();
+            cy.login();
+            cy.pbDeleteAllPages();
+            cy.pbCreatePage({ category: "static" }).then(page => {
+                cy.pbUpdatePage({
+                    id: page.id,
+                    data: {
+                        category: "static",
+                        path: `/page-${id}`,
+                        title: pageTitle1
+                    }
+                });
+                cy.pbPublishPage({ id: page.id });
+            });
+        });
+
+        describe("When Publishing new page", () => {
+            it("Step 1: Check page title in preview", () => {
+                cy.visit("/page-builder/pages");
+
+                // Check title on the page preview drawer.
+                cy.findByText(`${pageTitle1}`, { timeout: 15000 }).click({ force: true });
+                // Check whether we oppened a page preview drawer.
+                cy.url().should("include", "&id");
+                // Checking if title is correct.
+                cy.findByTestId("page-details-page-title").contains(`${pageTitle1}`);
+
+                // As we don't have a button to close page preview drawer,
+                // we need to simulate click outside of the drawer to close it.
+                cy.get("body").click("topLeft");
+                // Check if we closed page preview drawer.
+                cy.url().should("not.contain", "&id");
+
+                // Check title on the public site.
+                cy.pbListPages({ limit: 1, search: { query: pageTitle1 } }).then(pages => {
+                    pages.forEach(page => {
+                        cy.visit(`https://d37j6pfbndh1fp.cloudfront.net${page.path}`);
```

Comment:
Hardcoded URL here.

Reply:
Also, does this work for you? Not working over here. 

Cross-origin request-related error.

We did this by separating tests into multiple `it` blocks. That way you can test stuff on diff domains:
cypress/integration/admin/pageBuilder/menus/pagesListMenuItemType.spec.js:94

Reply:
Also take a look at this section that talks about caching:
https://github.com/webiny/webiny-js/blob/next/docs/CONTRIBUTING.md#how-to-test-appswebsite-app-in-the-cloud